bin/improver-weighted-blending

```diff
@@ -123,13 +128,24 @@ def main():
 
     cube = iris.load_cube(args.input_filepath)
 
+    if args.coord_exp_val:
```

Comment:
I've moved it into `process_coord`. Although I am passing `coord_value` straight into the `process` method, the CLI is still messy because I've added `--coordinate_units` and `--calender` optional arguments to deal with the coordinate units.